Article/Article.js

```diff
@@ -26,4 +33,12 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll(".article");
+// let article = document.querySelectorAll(".article");
+// let openArticle = document.querySelectorAll(".article-open");
+// let closeArticle = document.querySelectorAll(".close");
+
+
+articles.forEach(item => {
+  return new Article(item);
+});
```

Comment:
forEach doesn't return, you'll want to use a map for this for it's return functionality.  Don't forget you'll also need to assign it to something!